pkg/errutils/errutils.go

```diff
@@ -0,0 +1,14 @@
+package errutils
```

Comment:
wondering if we should put this in [internal](https://github.com/moby/moby/tree/master/internal) to reduce the number of pkg's we publish/have to maintain

(just a thought)